### ISSUE TITLE
compute_band fails on missing tables.index

### ISSUE THREAD

**Issue by BramSchnitzler** [openearth/flowmap#53]
* Flowmap generator version: 1.0.0
* Python version: 3.5
* Operating System: Windows

### Description

Tried to generate a subgridded waterdepth in .tiff format. The function 'compute_band' tried to get index from tables, but they were never created.

### What I Did

```
ugrid_dem = flowmap.formats.ugrid.UGrid(path = r'd:\Project\P940\05_Analysis\Resultaten\1131\Leerdam\leerdamwest_map.nc', dem = r'd:\Project\P940\05_Analysis\GIS\AHN\07_resultaat\C24\1131\resultaten\aw_ahn_def.tif')
t = 5
method = 'waterdepth'
format = '.tiff'


"""compute refined waterlevel using detailled dem, using subgrid or interpolate method"""
dem = read_dem(ugrid_dem.options['dem'])
grid = ugrid_dem.ugrid
data = ugrid_dem.waterlevel(t)

table_name = ugrid_dem.generate_name(
        ugrid_dem.path,
        suffix='.nc',
        topic='tables'
    )

table_path = pathlib.Path(table_name)

ugrid_dem.export('id_grid')
ugrid_dem.export('tables', valid_range = [-6,12]) 

tables = subgrid.import_tables(str(table_path))

band = subgrid.compute_band(grid, dem, tables, data, method=method)

Traceback (most recent call last):

  File "<ipython-input-17-ac79f7003f01>", line 1, in <module>
    band = subgrid.compute_band(grid, dem, tables, data, method=method)

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\site-packages\flowmap\subgrid.py", line 225, in compute_band
    faces = list(tables.index)

AttributeError: 'dict' object has no attribute 'index'
```

**Comment by SiggyF** [comment]
Can you use import_tables(..., arrays=False)? That should work. Import tables can return a pandas data frame or an array, by default it returns an array. I'll make this consistent so that we always use arrays. Using pandas for the bigger models was causing some issues.

**Comment by BramSchnitzler** [comment]
It works for the error mentioned above, but another error occured:
```

tables = subgrid.import_tables(str(table_path),arrays = False)
band = subgrid.compute_band(grid, dem, tables, data, method=method)

Traceback (most recent call last):

  File "<ipython-input-46-ac79f7003f01>", line 1, in <module>
    band = subgrid.compute_band(grid, dem, tables, data, method=method)

  File "d:\Project\P940\05_Analysis\Visualisatie\Flowmap\GitHub\flowmap\subgrid.py", line 244, in compute_band
    band[row['slice']] = result

  File "C:\Users\schnitzler\AppData\Local\Continuum\anaconda2_64\envs\P940_flowmap_05feb\lib\site-packages\numpy\ma\core.py", line 3301, in __setitem__
    _data[indx] = dval

ValueError: shape mismatch: value array of shape (4,4) could not be broadcast to indexing result of shape (4,4396)

```

**Comment by SiggyF** [comment]
w

**Comment by SiggyF** [comment]
accidently closed

**Comment by SiggyF** [comment]
Fixed in #56

**Comment by SiggyF** [comment]
In release v1.0.2
